tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,531 @@
+<tool
+  id="mqppep_preproc"
+  name="MaxQuant Phosphopeptide Preprocessing"
+  version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+  python_template_version="3.5"
+  profile="21.05"
+  >
+    <description>
+        Transform the output of MaxQuant for phosphoproteome-enriched samples to prepare it for statistical anlaysis.
```

Comment:
shorter please, not sentence

Reply:
I will try....

Reply:
This is still a sentence but hopefully short enough. I think that everything is necessary to understand what the tool is for:

"Prep phosphoproteomic MaxQuant output for statistical anlaysis."